packages/cloud-cognitive/src/components/Cascade/_cascade.scss

```diff
@@ -0,0 +1,50 @@
+//
+// Copyright IBM Corp. 2020, 2021
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+// Standard imports.
+@import '@carbon/import-once/scss/import-once';
+@import '../../global/styles/carbon-settings';
+@import '../../global/styles/project-settings';
+
+@mixin cascade {
+  $block-class: #{$pkg-prefix}--cascade;
+  $animationProps: 300ms cubic-bezier(0.2, 0, 0.38, 0.9) 0s 1;
```

Comment:
Does it make sense to reference some members from `@carbon/motion`'s Sass here?

Reply:
yeah i think so. sorry i should have marked this as a draft because i'm still getting input from the original developer 😅 

i'll probably be making some additional changes to this to get it in line